src/components/JobsTable/jobs-table.scss

```
@@ -100,6 +109,47 @@
     &__results {
       @include chip($resultsBorder);
     }
+    .row__actions {
+      display: none;
+      position: absolute;
+      right: -10px;
```

I think it looks better with:
```suggestion
      right: 0;
```